src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -1574,7 +1574,7 @@ def __init__(
                     num_attention_heads=temporal_num_attention_heads,
                     in_channels=out_channels,
                     num_layers=temporal_transformer_layers_per_block[i],
-                    norm_num_groups=temporal_norm_num_groups,
+                    norm_num_groups=resnet_groups,
```

Comment:
Is this a harmless change?

Reply:
I believe so, yes. All other occurrences of the implementation use `norm_num_groups=resnet_gruops` so this seems to be an uncaught bug for a long while. Relevant comment: https://github.com/huggingface/diffusers/pull/7707#discussion_r1579248808